fs-storage/tests/SyncStatus.java

```diff
@@ -0,0 +1,11 @@
+/// Represents the synchronization status of the storage.
+public enum SyncStatus {
```

Comment:
It's unfortunate that we cannot avoid duplicating the SyncStatus definition. I'd hoped the `jnix_macro` should would create a class definition as part of the exported package.